governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
-
-* Meet regularly, at least for 30 minutes every 3 weeks, except November and December
-* Keep up-to-date meeting notes, linked from the SIG's page in the community repo
-* Announce meeting agenda and minutes after each meeting, on their SIG mailing list
-* Record SIG meeting and make it publicly available
-* Ensure the SIG's mailing list and slack channel are archived
-* Report activity in the weekly community meeting at least once every 6 weeks
-* Participate in release planning meetings and retrospectives, and burndown meetings, as needed
-* Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc. 
-* Use the above forums as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
-* Represent the SIG for the PM group:
- * identify all features in the current release from the SIG
- * track all features (in the repo with all the fields complete)
- * attend your SIG meetings
- * attend the PM group meetings which occur 3-5 times per release
- * identify the annual roadmap
- * advise their SIG as needed
+Most of the above roles require continuous, significant involvement in the project. Kubernetes is a very
+high-volume project -- hundreds of PRs and issues per week, with contributors in more than a dozen
+timezones.
+
+If someone becomes unable or unwilling to continue in their roles, they may retire. If someone doesn't
+fulfill their role for 90 days or violates the code of conduct, they may be removed from the role
+(escalation/vote process TBD). If they wish to resume their role in the future, they may request to return
+to it by asking the current members filling that role.
+
+# Special Interest Group (SIG) Governance
+
+
+[SIGs](README.md#special-interest-groups-sig-and-working-groups) are the subteams of the project.
+SIGs own code and/or documentation of the project, share knowledge, bring new members up to speed, and
+more.
+
+We have a couple dozen SIGs, and contributors in more than a dozen timezones. People want to be able
+to follow what's going on without attending every meeting, which is impossible, especially for
+non-American timezones.
+
+In order to standardize Special Interest Group efforts, maximize transparency, and route contributors
+to the appropriate SIG, SIGs should follow the guidelines stated below:
+
+- Meet regularly by video conference, phone conference, or in person, at least for 30 minutes every month
+  except November and December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Announce meeting agenda before each meeting and post minutes after, on their SIG mailing list
+- Record SIG meeting and make it publicly available
+  - TBD: Some SIG leads have objected to this, and also pointed out that it’s generally less useful
```

Comment:
Given that the cost of recording is near-zero (at least for hangouts and zoom), perhaps we should leave it to the viewers/consumers of this information as to whether they prefer to watch the video or read the notes.  One benefit of recordings is that they are complete, which notes are often not.